author: Sune Vuorela <sune@debian.org>

Don't do debugging symbols for webkit on s390, it is too big for the linker.
Thanks to Oswald Buddenhagen for the hint.

Debian bug 528485

--- a/src/3rdparty/webkit/WebKit.pri
+++ b/src/3rdparty/webkit/WebKit.pri
@@ -43,6 +43,13 @@ CONFIG(release, debug|release) {
     DEFINES += NDEBUG
 }
 
+arch=$$system(dpkg-architecture -qDEB_BUILD_ARCH)
+equals(arch, s390) {
+	message("S/390 workaround: is not building with -g, but with -gstabs")
+	QMAKE_CXXFLAGS -= -g
+}
+
+
 BASE_DIR = $$PWD
 INCLUDEPATH += $$PWD/WebKit/qt/Api
 QMAKE_LFLAGS += -Wl,--version-script,$$BASE_DIR/symbols.filter
